### ISSUE TITLE
Why automate checking and installing latest ChromeDriver?

### ISSUE THREAD

**Issue by GCuser99** [dan-yok-jpn/SeleniumVBA#1]
This is interesting... 

But why are you automating installing the latest ChromeDriver when SeleniumVBA does this automatically for Edge, Chrome, and Firefox already? That is one of the nicest features of SeleniumVBA. You should never have to get the latest ChromeDriver. It defaults to placing the latest drivers in the Downloads folder but that can be changed using the SeleniumVBA.ini file.

See **Auto-Check-and-Install** section of [SeleniumVBA Wiki](https://github.com/GCuser99/SeleniumVBA/wiki#getting-started).

Cheers,

[GCuser99](https://github.com/GCuser99)

**Comment by dan-yok-jpn** [comment]
Thank you for pointing that out.

I created this workbook to distribute to Excel users who have no programming knowledge but who want to use WebScraiping.

Since even downloading ChromeDriver is a hassle for them, I designed it to be fixedly placed in %LOCALAPPDATA%/SeleniumVBA.

In this case, is it possible to delete lines 298-312 in Thisworkbook.cls?

**Comment by GCuser99** [comment]
Yes, you should not need to explicitly invoke WebDriverManager. Each time your code executes driver.StartChrome/driver.StartEdge, SeleniumVBA checks in user's Downloads folder for the driver, and if not there, or out of date (major version number is different than browser), then it will download the most recent driver to the Downloads folder. So check out your Downloads folder - you should see the latest drivers are there even though you are not using them!

If the Downloads folder is an acceptable place to store the drivers, then change `.StartChrome "%LOCALAPPDATA%\SeleniumVBA\chromedriver.exe"` in your script to simply `.StartChrome`.

If you want to continue to store drivers in "%LOCALAPPDATA%\SeleniumVBA\", then you have several options:

1. Use/Distribute SeleniumVBA.ini file with `auto_detect_and_update=False` and continue with your ThisWorkbook.cls current script as it is now. That way you won't be wasting unnecessary effort downloading drivers to the Downloads directory.
2. Use/Distribute SeleniumVBA.ini file with `auto_detect_and_update=True` and `driver_location_folder=%LOCALAPPDATA%\SeleniumVBA` and delete the lines in ThisWorkbook.cls that do the download.
3. If you do not want to utilize/distribute the Selenium.ini file, then use `.StartChrome "%LOCALAPPDATA%\SeleniumVBA\chromedriver.exe". SeleniumVBA should update the driver specified in that path. You can delete the lines in ThisWorkbook.cls that do the download.

Hope that helps.

**Comment by GCuser99** [comment]
I edited the post above.

**Comment by dan-yok-jpn** [comment]
Thank you for your guidance.

I rewrote ```ThisWorkbook.cls``` by adding a subroutine that changes the value of the ```driver_location_folder key``` in ```Selenium.ini```.

I have confirmed that it works correctly.
